docs/application/native/guides/multimedia/audio.md

```diff
@@ -1,17 +1,17 @@
 # Audio Management
 
 
-You can control the audio behavior of the application. You can control the output volume and sound session types, and query various information from sound devices. You can also use OpenAL to specify the objects and operations in producing high-quality audio output.
+You can control the audio behavior of the application. You can control the output volume and sound conflict scenario, and query various information from sound devices. You can also use OpenAL to specify the objects and operations in producing low-latency audio output.
```

Comment:
You can control the output volume and sound conflict scenarios. You can query various information from sound devices.

Please clarify, the above sentence can be rewritten in the following style also:
- You can also query various information about the output from the sound devices.

please correct my understanding, I wrote this with my limited knowledge.

Reply:
I can rewrite the sentence like the style you said
 - You can also query..

but I think various information "about the output" is not proper.
The information is consist of device type, name, id, and so on.
I'll update it soon again.